css/styles.css

```diff
@@ -0,0 +1,172 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  text-decoration: none;
+}
+
+:root {
+  --black: #1c1a19;
+  --wine: #730d15;
+  --red: #ff6b00;
+  --gray: gray;
+  --gray2: #262626;
+  --white: #f4f5f0;
+  --projectPurple: #6070ff;
+  --projPurpleLetsConnect: #7f8cff;
+  --projBackground: var(--white);
+  --projText: #344563;
+  --projTitleText: #172b4d;
+}
+
+body {
+  background-color: var(--projBackground);
+  width: 100%;
+  margin: 0;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  text-align: center;
+  color: var(--projectPurple);
+  margin-left: 24px;
+  min-width: 76px;
+}
+
+.logo:hover {
+  text-decoration: none !important;
+  color: var(--projectPurple);
+}
+
+.menu-hamburg {
+  color: var(--projectPurple);
+  width: 1fr;
+  margin-right: 20px;
+}
+
+header {
+  width: 100%;
+  height: 100px;
```

Comment:
- [ ] Kindly make sure to stick to the Figma design as much as possible as said in the requirements.
As you can compare your design :point_down: 
![Screenshot from 2022-09-23 17-10-38](https://user-images.githubusercontent.com/97100091/191999527-f3604eab-53f3-4899-8da7-086793695e9a.png)

To the Figma design provided :point_down: 
![Screenshot from 2022-09-23 17-37-14](https://user-images.githubusercontent.com/97100091/191999675-8c7a7064-d4f1-4b3d-b377-eb851cf1b809.png)
`hint: You may consider removing or reducing those **margins** right, left, and top on the _header_, _logo_, _hamburger menu_ and the _navbar_`

Reply:
It's ok.